src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -218,6 +224,30 @@ namespace OpenMS
     }
 
     /**
+     * @brief encapsulates readSpectra with file loading convenience if only a file path is given
+     * @param spectra_file file to the SpectrumContainer, must be valid path to mz file
+     */
+    void readMzFile(const String& spectra_file)
```

Comment:
name suggests that am MzML file is expected but the function is written for generic spectra files